packages/cli/src/sso/saml/routes/saml.controller.ee.ts

```diff
@@ -138,10 +138,10 @@ export class SamlController {
 				if (isSamlLicensedAndEnabled()) {
 					await issueCookie(res, loginResult.authenticatedUser);
 					if (loginResult.onboardingRequired) {
-						return res.redirect(getInstanceBaseUrl() + SamlUrls.samlOnboarding);
+						return res.redirect(this.urlService.editorBaseUrl + SamlUrls.samlOnboarding);
```

Comment:
this should be using `editorBaseUrl` so that these urls work tunnels, and also when someone is overriding the base url.